src/libs/curated_app_libs.py

```diff
@@ -208,7 +205,6 @@ def run_test(test_instance, test_yaml_file):
                 return result
         
         if "expected_output_console" in test_config_dict.keys():
-            cleanup_after_test(workload_name)
```

Comment:
and this too?

Reply:
this will be taken care in finally statement...

Reply:
same here also.

Reply:
what did we decide? Even if thought it will return before hitting finally, does it run finally and then return?

Reply:
finally block will be executed even after return statement execution